src/content/app/genome-browser/state/track-settings/trackSettingsConstants.ts

```diff
@@ -27,6 +27,7 @@ import type {
 export enum TrackType {
   GENE = 'gene',
   FOCUS_GENE = 'gene-focus',
+  FOCUS_VARIANT = 'focus-variant',
```

Comment:
Either of this could be renamed to match the prefix.

Reply:
Changed to `'focus-gene'`